mappings/net/minecraft/world/gen/feature/SurfacePlacedFeatures.mapping

```diff
@@ -0,0 +1,14 @@
+CLASS net/minecraft/class_6825 net/minecraft/world/gen/feature/SurfacePlacedFeatures
```

Comment:
Since all usages of this class is in `*VillageData`, it's probably village related features.